modules/installation-mirror-repository.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-mirror-repository_{context}"]
+= Mirroring the {product-title} image repository
+
+Mirror the {product-title} image repository to use during cluster installation
+or upgrade.
+
+.Prerequisites
+
+* You prepared the bastion host to use in a disconnected installation.
+* You configured a mirror registry to use in your disconnected environment and
+can access the certificate and credentials that you configured.
+* You downloaded the pull secret from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
+page and modified it to include authentication to your mirror repository.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Review the
+link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Quay.io]
+repository to determine the version of {product-title} that you want to install.
+
+. Set the required environment variables:
++
+----
+$ export OCP_RELEASE=<release_version> <1>
+$ export AIRGAP_REG='<bastion_host_name>:5000' <2>
+$ export AIRGAP_REPO='<repo_name>' <3>
+$ export PRODUCT_REPO='openshift-release-dev' <4>
+$ export AIRGAP_SECRET_JSON='<path_to_pull_secret>${HOME}/pull-secret-2.json' <5>
+$ export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${AIRGAP_REG}/${AIRGAP_REPO}:${OCP_RELEASE} <6>
+$ export RELEASE_NAME="ocp-release" <7>
+----
+<1> For `<release_version>`, specify the version number of {product-title} to
+install, such as `4.2.0`.
+<2> For `<bastion_host_name>`, specify the registry domain name for your mirror
+repository.
+<3> For `<repo_name>`, specify the name of the repository to create in your
+registry. Use a descriptive name, such as `ocp4/openshift4`.
```

Comment:
nit: "Use a descriptive name" seems overly prescriptive.  Maybe just "...in your registry, such as `ocp4/openshift4`."